packages/edit-post/src/hooks/commands/use-common-commands.js

```diff
@@ -152,4 +169,56 @@ export default function useCommonCommands() {
 			openModal( KEYBOARD_SHORTCUT_HELP_MODAL_NAME );
 		},
 	} );
+
+	useCommand( {
+		name: 'core/toggle-breadcrumbs',
+		label: showBlockBreadcrumbs
+			? __( 'Hide block breadcrumbs' )
+			: __( 'Show block breadcrumbs' ),
+		icon: cog,
+		callback: ( { close } ) => {
+			toggle( 'core/edit-post', 'showBlockBreadcrumbs' );
+			close();
+			createInfoNotice(
+				showBlockBreadcrumbs
+					? __( 'Breadcrumbs off.' )
+					: __( 'Breadcrumbs on.' ),
+				{
+					id: 'core/edit-post/toggle-breadcrumbs/notice',
+					type: 'snackbar',
+				}
+			);
+		},
+	} );
+
+	useCommand( {
+		name: 'core/toggle-publish-sidebar',
+		label: isPublishSidebarEnabled
+			? __( 'Disable pre-publish checklist' )
+			: __( 'Enable pre-publish checklist' ),
+		icon: formatListBullets,
+		callback: ( { close } ) => {
+			close();
+			toggle( 'core/edit-post', 'isPublishSidebarEnabled' );
+			createInfoNotice(
+				isPublishSidebarEnabled
+					? __( 'Pre-publish checklist off.' )
+					: __( 'Pre-publish checklist on.' ),
+				{
+					id: 'core/edit-post/publish-sidebar/notice',
+					type: 'snackbar',
+				}
+			);
+		},
+	} );
+
+	useCommand( {
+		name: 'core/preview-link',
+		label: __( 'Preview in a new tab' ),
+		icon: external,
+		callback: ( { close } ) => {
+			close();
+			window.open( previewLink || currentPostLink, '_blank' );
+		},
+	} );
```

Comment:
This command will need a few adjustments. It needs to dispatch the `__unstableSaveForPreview` action so that changes are reflected in previews.

Reply:
Good point. If you have time, feel free to update. 🚀

Reply:
Done: [4878398](https://github.com/WordPress/gutenberg/pull/53073/commits/4878398b1b35c7348cfa42eee6e595da6456fe04).